src/types/posts.js

```diff
@@ -43,7 +48,7 @@ export type PostMetadata = {|
     reactions: Array<Reaction>
 |};
 
-export type Post = {|
+export type Post = {
```

Comment:
@jespino Are you okay with this change? This is what I mentioned yesterday where Post being an exact type prevented PostWithFormatData from being passed into functions like `isSystemMessage`

Reply:
Yes, that works for me, is not "super-strict" about typing, but I think is a good solution. We can revisit it later when the whole thing is in Flow.